src/Microsoft.AspNet.TestHost/HttpResponseMessageExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.TestHost
+{
+    public static class HttpResponseMessageExtensions
+    {
+        public static Task PipelineCompleteAsync(this HttpResponseMessage message)
```

Comment:
This seems overkill. Why can't the client just wait for the end of the response stream?